includes/admin/settings/class-wcmp-shipping-methods.php

```diff
@@ -40,8 +42,11 @@ class WCMP_Shipping_Methods
      */
     private $shippingMethods = [];
 
+    private $shippingZones;
```

Comment:
phpdoc mist

Reply:
Je moet aangeven wat erin zit, `array` zegt niet genoeg. De comment zou niet nodig moeten zijn (en bevat fouten).

Het moet `string[]` of `int[]` of iets dergelijks zijn, uiteraard afhankelijk van wat je erin stopt. Zo te zien zijn het ids, dus misschien moet je hem zelfs hernoemen naar `$shippingZoneIds`